app/flows/check_travel_during_coronavirus_flow/outcomes/_fully_vaxed.erb

```diff
@@ -12,17 +12,35 @@
 
   <p class="govuk-body">You do not need to take any COVID-19 tests before you travel to England or after you arrive.</p>
 
-  <p class="govuk-body">You must be able to prove that you’ve been fully vaccinated. If you live in England, you can prove your vaccination status using the <a href="https://www.gov.uk/guidance/nhs-covid-pass" class="govuk-link">NHS COVID Pass</a>.</p>
+  <p class="govuk-body">You must <a href="https://www.gov.uk/provide-journey-contact-details-before-travel-uk" class="govuk-link">complete a passenger locator form</a> in the 48 hours before you arrive in England.</p>
+
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Proof of vaccination",
+    heading_level: 3,
+    font_size: "m",
+    margin_bottom: 4,
+  } %>
+
+  <p class="govuk-body">You must be able to prove that you've been fully vaccinated. If you live in England, you can prove your vaccination status using the <a href="https://www.gov.uk/guidance/nhs-covid-pass" class="govuk-link">NHS COVID Pass</a>.</p>
```

Comment:
I think there's some missing text here, missing curly quotes and another absolute like that should be relative:
```suggestion
  <p class="govuk-body">You must be able to prove that you’ve been fully vaccinated against COVID-19. If you live in England, you can prove your vaccination status using the <a href="/guidance/nhs-covid-pass" class="govuk-link">NHS COVID Pass</a>.</p>
```

Reply:
The content changes in this view partial don't seem to be related to the rest of the "Because you said" changes. I think they have a separate Trello card? You could separate them out into a different PR to get them approved more quickly.

Reply:
Nice one, sorted. I'll leave it in this PR because I don't think we're a million miles away